src/WcfCoreMtomEncoder/MtomPart.cs

```diff
@@ -32,7 +45,8 @@ public byte[] GetRawContent()
 
         public string GetStringContentForEncoder(MessageEncoder encoder)
         {
-            if (!ContentType.Parameters.Any(p => p.Name == "type" && encoder.IsContentTypeSupported(p.Value.Replace("\"", ""))))
+            if (ContentType == null || 
```

Comment:
For readability sake, I'd prefer braces around the if body when the conditional spans multiple lines

Reply:
Sure!